c++/src/capnp/serialize.h

```diff
@@ -53,7 +53,14 @@ class FlatArrayMessageReader: public MessageReader {
 
 public:
   FlatArrayMessageReader(kj::ArrayPtr<const word> array, ReaderOptions options = ReaderOptions());
-  // The array must remain valid until the MessageReader is destroyed.
+  // The array must remain valid until the MessageReader is destroyed. Example:
+  //
+  //     auto fs = kj::newDiskFilesystem();
+  //     auto file = fs->getCurrent().openFile(kj::Path::parse("mydata"));
+  //     kj::Array<const kj::byte> fileBuffer = file->mmap(0, file->stat().size).releaseAsChars()
+  //     kj::ArrayPtr<const capnp::word> messageBuffer(
+  //       reinterpret_cast<const capnp::word *>(fileBuffer.begin()), fileBuffer.size() / sizeof(capnp::word));
+  //     capnp::FlatArrayMessageReader capnpReader { messageBuffer };
```

Comment:
Braced initializers for non-structs weird me out, and you used a parenthesized initializer on the previous line anyway, so let's be consistent.

```suggestion
  //     capnp::FlatArrayMessageReader capnpReader(messageBuffer);
```